autogen/AudioSource.m

```diff
@@ -0,0 +1,78 @@
+% AudioSource 
+%
+% This class represents an audio source for use with Autogen.
+%
+% Copyright 2019 Audio Logic
+%
```

Comment:
add a space between the description and the copyright so the copyright/license don't show up in the help documentation

Reply:
Done